truffle/src/main/java/org/truffleruby/parser/BodyTranslator.java

```diff
@@ -1729,7 +1729,7 @@ public RubyNode visitGlobalAsgnNode(GlobalAsgnParseNode node) {
             rhs = new CheckOutputSeparatorVariableTypeNode(rhs);
             rhs.unsafeSetSourceSection(sourceSection);
         } else if (name.equals("$_")) {
-            if (getSourcePath(sourceSection).endsWith(buildPartialPath("truffle", "rubysl", "rubysl-stringio", "lib", "rubysl", "stringio", "stringio.rb"))) {
+            if (getSourcePath(sourceSection).endsWith(buildPartialPath("rubysl", "rubysl-stringio", "lib", "rubysl", "stringio", "stringio.rb"))) {
```

Comment:
I do see that this code is just replacing the path being used. I'm just raising as FYI and partially in response to Benoit's suggestion.